### PR TITLE
chinese translation

### PR DESCRIPTION
**Pull request by Manasa2850**
# Description
< Added files for chinese translation>

## Corresponding Issue

< https://github.com/ifmeorg/ifme/issues/1345 >

### CODE REVIEW

**Comment by Manasa2850** [comment]
@julianguyen Yes I will be continuing it.

File: config/locales/de.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@ de:
     de: Deutsch
     fr: Français
     hi: 'हिंदी'
+    ch: 中文
```
Comment (julianguyen) [inline config/locales/de.yml]: I looked up the language code for Chinese and it's actually `zh`.

Also what do you think of specifying whether it's Simplified or Traditional?


Chinese | zh
Chinese (Simplified) | zh-Hans
Chinese (Traditional) | zh-Hant

Comment (Manasa2850) [reply config/locales/de.yml]: @julianguyen sure I'll do that!
I also need separate yml files (like ch.yml) for the three right?

Comment (julianguyen) [reply config/locales/de.yml]: Yeah you'll want to create the following [here](https://github.com/ifmeorg/ifme/tree/master/config/locales):
* zh.yml
* devise.zh.yml
* devise_invitable.zh.yml
* zh.yml
* kaminari.zh.yml

Not sure if you've already seen our [Translations guide](https://github.com/ifmeorg/ifme/wiki/Translations)!

Comment (Manasa2850) [reply config/locales/de.yml]: @julianguyen Thanks! I'll make changes and submit the final PR in a day or two!

**Pull request merged**